src/styles/blocks/cards.css

```diff
@@ -0,0 +1,92 @@
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 4px;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  margin: 32px auto 40px;
+}
+
+.card__description {
+  padding: 0 16px;
```

Comment:
Let's add general paddings to the whole product card, instead of adding them to the each element